prototype/Frontend/admin-Sign-in.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Admin Login / Sign Up Form</title>
+    <link rel="stylesheet" href="css/admin-login.css" />
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header>
+      <a href="index.html" id="logo"
+        ><img src="/assets/blogs/blog_img_6.jpeg" alt="Logo"
+      /></a>
+      <a id="website-name">THE WEEKLY CHALLENGE - PERL and RAKU </a>
+    </header>
+
+    <!-- Main Content Section -->
+    <div class="container">
+      <div class="forms-container">
+        <!-- Sign-up Form -->
+        <div class="form-control signup-form">
+          <form action="#">
+            <h2>Admin Sign-up</h2>
+            <input type="text" placeholder="Username" required />
+            <input type="email" placeholder="Email" required />
+            <input type="password" placeholder="Password" required />
+            <input type="password" placeholder="Confirm password" required />
+            <div class="captcha-container">
+              <img id="captcha-image" src="#" alt="CAPTCHA Image" />
+
+              <div
+                style="
+                  display: flex;
+                  justify-content: space-evenly;
+                  align-items: center;
+                "
+              >
```

Comment:
Again this should have been:

        <div  style="display: flex; justify-content: space-evenly; align-items: center;">